modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,441 @@
+$ rosa create admin --cluster=<clusterName>
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster to add to the identity provider (IDP).
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+Create a cluster administrator that can log into a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa create admin --cluster-name=mycluster
+----
+
+[id="rosa-create-cluster_{context}"]
+== create cluster
+
+Creates a new cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create cluster --cluster=<clusterName> [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster. This is used when generating a sub-domain for your cluster on openshiftapps.com.
+
+|--channel-group
+|The channel group (string) is the name of the group where this image belongs, for example _stable_ or _fast_. Default: stable
+
+|--compute-machine-type
+|The instance type (string) for the compute nodes. Determines the amount of memory and vCPU allocated to each compute node.
+
+|--compute-nodes
+|The number (integer) of worker nodes to provision per zone. Single zone clusters require at least 2 nodes. Multi-zone clusters require at least 3 nodes. Default: 2
+
+|--disable-scp-checks
+|Indicates whether cloud permission checks are disabled when attempting the installation of the cluster.
+
+|--dry-run
+|Simulates creating the cluster.
+
+|--host-prefix
+|The subnet prefix length (integer) to assign to each individual node. For example, if host prefix is set to _23_, then each node is assigned a /23 subnet out of the given CIDR.
+
+|--machine-cidr
+|Block of IP addresses (ipNet) used by OpenShift while installing the cluster. Example: 10.0.0.0/16
+
+|--multi-az
+|Deploys to multiple data centers.
+
+|--pod-cidr
+|Block of IP addresses (ipNet) from which Pod IP addresses are allocated. Example: 10.128.0.0/14
+
+|--private
+|Restricts master API endpoint and application routes to direct, private connectivity.
+
+|-r, --region
+|The AWS region (string) where your worker pool will be located. This argument overrides the AWS_REGION environment variable.
+
+|--service-cidr
+|Block of IP addresses (ipNet) for services. Example: 172.30.0.0/16
+
+
+|--subnet-ids
+|The subnet IDs (string) to use when installing the cluster. SubnetIDs must be in pairs with one private subnetID and one public subnetID per availability zone. Subnets are comma delimited. Example: ``--subnet-ids=subnet-1,subnet-2`. Leave the value empty for installer provisioned subnet IDs.
+
+|--version
+|The version (string) of OpenShift that will be used to install the cluster. Example: 4.3.10
```

Comment:
Same comment about not hard-coding "OpenShift"

Reply:
Can't be changed at this time due unless something can be changed in how we use the attributes file.